src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -168,6 +168,11 @@ pub struct Args {
     #[clap(long, env = "UNSAFE_MODE")]
     unsafe_mode: bool,
 
+    /// Enables features appropriate only for local development, such as
+    /// enabling all optional features.
+    #[clap(long, env = "LOCAL_DEV")]
+    local_dev: bool,
```

Comment:
Wait, what actually _uses_ this?

Reply:
I thought it was used for working on the web stuff so it'd use local files instead of compiled, to avoid recompilation during change.